pkgs/development/libraries/openldap/default.nix

```diff
@@ -54,19 +56,20 @@ stdenv.mkDerivation rec {
     "moduledir=$(out)/lib/modules"
   ];
 
-  # 1. Fixup broken libtool
-  # 2. Libraries left in the build location confuse `patchelf --shrink-rpath`
+  # 1. Libraries left in the build location confuse `patchelf --shrink-rpath`
   #    Delete these to let patchelf discover the right path instead.
   #    FIXME: that one can be removed when https://github.com/NixOS/patchelf/pull/98
```

Comment:
Is that PR already in nixpkgs? Maybe we should address that FIXME some time.

Reply:
No that PR is not yet in nixpkgs it seems, tried removing the lines, issue still stands.

Reply:
Correction: the feature is in `nixpkgs` but the `patchelf` setup-hook doesn't use the `allowed-rpath-prefixes` option and there's no way to pass it as well.